dotnet/src/SemanticKernel.Core/Extensions/KernelSemanticFunctionExtensions.cs

```diff
@@ -240,34 +266,36 @@ public static class KernelSemanticFunctionExtensions
                 if (!File.Exists(promptPath)) { continue; }
 
                 // Load prompt configuration. Note: the configuration is optional.
-                var config = new PromptTemplateConfig();
+                var promptTemplateConfig = new PromptTemplateConfig();
                 var configPath = Path.Combine(dir, ConfigFile);
                 if (File.Exists(configPath))
                 {
-                    config = PromptTemplateConfig.FromJson(File.ReadAllText(configPath));
+                    promptTemplateConfig = PromptTemplateConfig.FromJson(File.ReadAllText(configPath));
```

Comment:
```C#
var configPath = Path.Combine(dir, ConfigFile);
var promptTemplateConfig = File.Exists(configPath) ?
    PromptTemplateConfig.FromJson(File.ReadAllText(configPath)) :
    new PromptTemplateConfig();
```
to avoid the unnecessary allocation when the file doesn't exist.